app/controllers/vatapplication/TotalTaxTurnoverEstimateController.scala

```diff
@@ -42,11 +42,13 @@ class TotalTaxTurnoverEstimateController @Inject()(val sessionService: SessionSe
   val show: Action[AnyContent] = isAuthenticatedWithProfile {
     implicit request =>
       implicit profile =>
-        vatApplicationService.getVatApplication.map {
-          vatApp => {
-            Ok(view(formProvider().fill(vatApp.acceptTurnOverEstimate.fold(false)(fg=>fg)), getAmount(vatApp.standardRateSupplies),
-              getAmount(vatApp.reducedRateSupplies), getAmount(vatApp.zeroRatedSupplies), getAmount(vatApp.turnoverEstimate)))
-          }
+        vatApplicationService.getVatApplication.map { rec =>
```

Comment:
Do we have UT's that test to see if the page displays 'yes' , 'no' or no content versions depending on different user data?